.github/workflows/test.yml

```diff
@@ -9,6 +9,23 @@ on:
       - '.gitignore'
 
 jobs:
+  tests:
+    runs-on: ubuntu-latest
+    container:
+      image: python:3.12
```

Comment:
🐍 Using Python 3.12 is a good choice for leveraging the latest features and improvements!